src/connectors/scenes/FioAddressConfirmConnector.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+
+import { type EdgeCurrencyWallet } from 'edge-core-js'
+import { connect } from 'react-redux'
+
+import { createCurrencyWallet } from '../../actions/CreateWalletActions.js'
+import type { DispatchProps, StateProps } from '../../components/scenes/FioAddressConfirmScene'
+import { FioAddressConfirmScene } from '../../components/scenes/FioAddressConfirmScene'
+import { getAccount } from '../../modules/Core/selectors'
+import { changeConfirmSelectedWallet } from '../../modules/FioAddress/action'
+import { getFioWallets } from '../../modules/UI/selectors'
+import type { Dispatch, State } from '../../types/reduxTypes'
+
+const mapStateToProps = (state: State) => {
+  const { fioAddress } = state.ui.scenes
+  const fioWallets: EdgeCurrencyWallet[] = getFioWallets(state)
+  const account = getAccount(state)
+
+  const out: StateProps = {
+    fioAddressName: fioAddress.fioAddressName,
+    account,
+    fioWallets
+  }
+  return out
+}
+
+const mapDispatchToProps = (dispatch: Dispatch): DispatchProps => ({
+  createCurrencyWallet: (walletName: string, walletType: string) => dispatch(createCurrencyWallet(walletName, walletType, 'iso:USD', false, false)),
```

Comment:
Should not hardcode `iso:USD`. Look up the fiat currency of the account to determine which fiat currency to use for the wallet. Even better, let the user choose.

Reply:
get default value for currency